protocols/kad/src/behaviour.rs

```diff
@@ -23,6 +23,7 @@
 mod test;
 
 use crate::addresses::Addresses;
+use crate::bootstrap::Status;
```

Comment:
Since there is already a `use crate::{bootstrap, K_VALUE};`

```suggestion
```

Reply:
Yes please.